packages/calling-stateful-client/src/SpotlightSubscriber.ts

```diff
@@ -33,10 +33,13 @@ export class SpotlightSubscriber {
     this._spotlightFeature.off('spotlightChanged', this.spotlightChanged);
   };
 
-  private spotlightChanged = (): void => {
+  private spotlightChanged = (args: { added: SpotlightedParticipant[]; removed: SpotlightedParticipant[] }): void => {
     this._context.setSpotlight(this._callIdRef.callId, this._spotlightFeature.getSpotlightedParticipants());
-    for (const addedParticipant of this._spotlightFeature.getSpotlightedParticipants()) {
+    for (const addedParticipant of args.added) {
       this._context.setParticipantSpotlighted(this._callIdRef.callId, addedParticipant);
     }
+    for (const removedParticipant of args.removed) {
+      this._context.setParticipantNotSpotlighted(this._callIdRef.callId, removedParticipant);
+    }
```

Comment:
I was not removing spotlight form RemoteParticipantState but I got away with it because I was using spotlightedParticipants from SpotlightCallFeatureState to know who is spotlighted in VideoGallery and ParticipantList